contained-innersource-enables-collaborative-product-development.md

```diff
@@ -0,0 +1,35 @@
+## Title  
+Contained inner source enables collaborative product development
+
+## Context
```

Comment:
Is there a previously written component of is this a green field situation? I think it matters in the case where one of the BUs is the current owner of the component.